docs/source/ROL.rst

```diff
@@ -105,21 +104,42 @@ method.
     situations when the routine that evaluates the functional :bash:`J`
     fails. Such failures are usually due to failure in solving the state
     constraint. Among other reasons, this can happen when the control
-    :bash:`q` is not feasible (for instance, when the underlying mesh interstects itself)
-    or when the state constraint is nonlinear and the optimization step
-    is too large (in which case the initial guess is not good enough).
+    :bash:`q` is not feasible (for instance, when the underlying mesh
+    interstects itself) or when the state constraint is nonlinear and
+    the optimization step is too large (in which case the initial guess
+    is not good enough).
+
+    In a nutshell, trust-region methods solve a sequence of optimization
+    problems. In each of these, one minimizes a quadratic misfit functional
```

Comment:
I don't see where the _misfit_ functional comes from. Do you just mean a quadratic approximation to the true objective? I associate the term _misfit_ with something like a l^2 minimization, which we don't necessarily do.

Reply:
I dropped misfict and, when necessary, replaced it with objective